.editorconfig

```diff
@@ -9,5 +9,4 @@ trim_trailing_whitespace = true
 charset = utf-8
 
 [{*.json,.*rc,*.yml}]
-indent_style = space
```

Comment:
Please don't remove this

Reply:
But, doesn't redundante in `[*]`?

Reply:
Thats a good point. Didn't think of that. Looks like its really redundant